### PR TITLE
Update JetBrains docs (mainly updating GIFS)

### PR DESCRIPTION
**Pull request by loujaybee**
Various updates to JetBrains documentation

- Move FAQs to dedicated IDE's section (from troubleshooting)
- Update IDE pages with 3 up-to-date GIF's (removed the 1 old one)
- Added some additional details / notes added to the JetBrains Gateway dedicated page

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1929"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1929"></a>

### CODE REVIEW

File: gitpod/docs/ides-and-editors/faqs.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Can I use both a desktop IDE and a browser editor in Gitpod?
+
+Yes, you can. Both desktop and browser IDEs/editors have full access to files and directories within the workspace. So, when you connect to your Gitpod workspace from your Desktop you can also access your workspace via your browser. This means browser-based editing can either be used as your first-choice editor, or as a "fall back" alternative option for when you want to make edits on the move. With Gitpod, where and how you edit your code is very flexible.
+
+## What is the preferred IDE/editor for Gitpod?
+
+There are many ways to edit your code in Gitpod, using [supported JetBrains IDE's](/docs/ides-and-editors) via [JetBrains Gateway](/docs/ides-and-editors/jetbrains-gateway), or by using [VS Code in the browser](/docs/ides-and-editors/vscode-browser) / [VS Code on desktop](/docs/ides-and-editors/vscode).
```
Comment (akosyakov) [inline gitpod/docs/ides-and-editors/faqs.md:26]: can mention SSH now

File: gitpod/docs/ides-and-editors/jetbrains-gateway.md [hunk 2]
```diff
@@ -18,9 +18,49 @@ Using JetBrains Gateway on Desktop with Gitpod, allows you to:
...
+You can also start, and connect to Gitpod workspaces directly from JetBrains Gateway. Click Gitpod in the left-hand side of JetBrains Gateway, and paste a source control context URL directly to start a new workspace, or click connect on a running or stopped workspace.
+
+<figure>
+<img class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Selecting IntelliJ in the preferences" src="/images/editors/open-from-jetbrains-gateway.gif">
+    <figcaption>Opening a JetBrains IDE from Gateway</figcaption>
+</figure>
+
+### How does JetBrains and Gitpod work?
+
+On Gitpod workspace start, a JetBrains IDE is loaded into your workspace according to your preferences. When you start a workspace the JetBrains Gateway application downloads a thin client which then connects to your Gitpod workspace.
```
[the remark this answers is not in the document]
Comment (loujaybee) [reply gitpod/docs/ides-and-editors/jetbrains-gateway.md]: Yeah, I was toying with that myself, pulled up top! f160e8725d74aa5bbe339058d94d80fe67180816

File: gitpod/docs/ides-and-editors/faqs.md (hunk 1 above)
Comment (loujaybee) [reply gitpod/docs/ides-and-editors/faqs.md]: True! Added d5ad96a61331016e651d043c69b7cfe4638f2780

**Review by akosyakov — APPROVED** [review]
LGTM

Maybe highlight `you do not need to download the full JetBrains IDEs themselves` in. `Since JetBrains Gateway downloads the thin client for you, you do not need to download the full JetBrains IDEs themselves.` as well.

**Comment by loujaybee** [comment]
@akosyakov - done!

**Pull request merged**